src/index.html

```diff
@@ -1,13 +1,427 @@
+    <link 
+      rel="shortcut icon" 
+      href="../src/images/logo.png" 
+      type="image/x-icon"
+    >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com">
+    <link  
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto&display=swap" 
+      rel="stylesheet"
+    >
+
+    <link 
+      rel="stylesheet" 
+      href="./styles/main.scss"
+    >
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#Home" class="logo">
+              <img 
+                class="logo__image"
+                src="./images/logo.svg"
+                alt="THE MET"
+              >
+            </a>
+            <div class="header__bottom">
+              <a href="tel:+1 212-535-7710" class="icon icon--call header__call"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <h1 class="header__title">Welcome <br> to The MET</h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="menu page__menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="#Home" class="logo ">
+              <img 
+                class="logo__image"
+                src="./images/logo.svg"
+                alt="THE MET"
+              >
+            </a>
+
+            <a href="#" class="icon icon--cross"></a>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#Home" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#About Us" class="menu__link">Museum hours</a>
+            </li>
+            <li class="menu__item">
+              <a href="#Now on view" class="menu__link">Now on view</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">Explore Online</a>
+            </li>
+            <li class="menu__item">
+              <a href="#Contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a 
+            href="tel:+1 212-535-7710" 
+            class="menu__call menu__call--fifth"
+          >
+            The Met Fifth Avenue <br> 
+            +1 212-535-7710
+          </a>
+          <a 
+            href="tel:+1 212-923-3700" 
+            class="menu__call menu__call--cloisters"
+          >
+            The Met Cloisters <br>
+            +1 212-923-3700
+          </a>
+          <a href="#call to order" class="menu__action menu__action--order">
+            call to order
+          </a>
```

Comment:
Add some hover effects here, also `call to order` text looks a bit too small
![image](https://user-images.githubusercontent.com/37161315/170650943-5be67999-b2dd-496b-bd23-a3501eff1a07.png)